test/common/fixtures.go

```diff
@@ -42,28 +41,20 @@ func TypeConfigFixturesOrDie(tl TestLogger) map[string]*unstructured.Unstructure
 }
 
 func typeConfigFixtures() (map[string]*unstructured.Unstructured, error) {
-	path := fixturePath()
-	files, err := ioutil.ReadDir(path)
-	if err != nil {
-		return nil, errors.Wrapf(err, "Error reading fixture from path %q", path)
-	}
-
 	fixtures := make(map[string]*unstructured.Unstructured)
 	suffix := ".yaml"
-	for _, file := range files {
-		if !strings.HasSuffix(file.Name(), suffix) {
-			continue
-		}
-
-		typeConfigName := strings.TrimSuffix(file.Name(), suffix)
-
-		filename := filepath.Join(path, file.Name())
+	prefix := "test/common/fixtures/"
+	for _, file := range AssetNames() {
 		fixture := &unstructured.Unstructured{}
-		err := kfenable.DecodeYAMLFromFile(filename, fixture)
-		if err != nil {
-			return nil, errors.Wrapf(err, "Error reading fixture for %q", typeConfigName)
+		if strings.HasPrefix(file, prefix) {
```

Comment:
It's generally preferable to minimize the scope of indentation where possible.  In this case, that would mean doing the following at the top of the loop block:

```golang
if !strings.HasPrefix(...) {
  continue
} 
...
```